front/app/components/CityLogoSection/index.tsx

```diff
@@ -65,7 +65,11 @@ const CityLogoSection = ({
 
       return (
         <Fragment
-          title={formatMessage(messages.iframeTitle)}
+          title={formatMessage(messages.iframeTitle, {
+            orgName: localize(
+              appConfiguration.attributes.settings.core.organization_name
+            ),
```

Comment:
Just the comment I had from the other day - should we put `localize(
              appConfiguration.attributes.settings.core.organization_name
            ),` into a util somewhere to make this more concise/easier to change later if needed? :)

Reply:
What kind of util are you thinking of? Could you make a PR to this branch with a suggestion?

Reply:
Hmm - I was thinking of something in the helperUtils, but you can't use localize in a function right? We don't seem to have any other functions that do that either - so maybe better to leave it as you have it actually :)

Reply:
Yeah, I guess that's why we had orgName as part of our custom formatMessage/FormattedMessage, but I had a type issue I couldn't easily fix + it was a bit obscure imo. I guess it wasn't clear to everyone that you had these vars available.